langwatch/src/server/mailer/usageLimitEmail.tsx

```
@@ -0,0 +1,385 @@
+                    fontWeight: 600,
+                    color: "#6b7280",
+                    textTransform: "uppercase",
+                    letterSpacing: "0.5px",
+                  }}
+                >
+                  MESSAGES
+                </th>
+              </tr>
+            </thead>
+            <tbody
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)